src/FsAutoComplete.Core/AdaptiveExtensions.fs

```diff
@@ -369,7 +386,7 @@ and AdaptiveCancellableTask<'a>(cancel: unit -> unit, real: Task<'a>) =
   /// <summary>Will run the cancel function passed into the constructor and set the output Task to cancelled state.</summary>
   member x.Cancel() =
     cancel ()
-    cts.Cancel()
+    cts.TryCancel()
```

Comment:
We don't care if the cancellation can't be cancelled anymore. Alternative approaches would be to swap the cancel function out upon finishing of a task but this seemed easier.